### PR TITLE
Add byte conversion to allow int NBT values for Fireworks

### PR DESCRIPTION
**Pull request by rtm516**
Fixes https://github.com/GeyserMC/Geyser/issues/679

### CODE REVIEW

**Comment by Arktisfox** [comment]
Would `Integer.valueOf( nbt tag ).byteValue();` work better here, instead of converting to string and back?

**Comment by rtm516** [comment]
> Would `Integer.valueOf( nbt tag ).byteValue();` work better here, instead of converting to string and back?

No because it is sometimes a byte and sometimes an int, whereas that only takes int or string, so it would need a `toString` again

**Comment by Arktisfox** [comment]
Tested on Paper with the commands listed in the issue, appears to be fixed (both entity and item forms)

File: connector/src/main/java/org/geysermc/connector/utils/MathUtils.java [hunk 1]
```diff
@@ -37,4 +37,17 @@ public static int ceil(float floatNumber) {
...
     }
+
+    /**
+     * Converts the given object from an int or byte to byte
+     *
+     * @param value The value to convert
+     * @return The converted byte
+     */
+    public static Byte convertByte(Object value) {
+        if(value instanceof Integer){
```
Comment (DoctorMacc) [inline connector/src/main/java/org/geysermc/connector/utils/MathUtils.java:48]: Spacing

Comment (DoctorMacc) [reply connector/src/main/java/org/geysermc/connector/utils/MathUtils.java:48]: Additionally - perhaps a tiny explanation of why we need to do this?

**Pull request merged**